client/src/app/pages/identities/identities.tsx

```diff
@@ -317,16 +292,27 @@ export const Identities: React.FC = () => {
             }
           />
         </ConditionalRender>
-        <NewIdentityModal
-          isOpen={isIdentityModalOpen}
-          onSaved={handleOnIdentityCreated}
-          onCancel={closeIdentityModal}
-        />
-        <UpdateIdentityModal
-          identity={rowToUpdate}
-          onSaved={handleOnIdentityUpdated}
-          onCancel={handleOnCancelUpdateIdentity}
-        />
+
+        <Modal
+          title={
+            identityToUpdate
+              ? t("dialog.title.update", {
+                  what: t("terms.credential").toLowerCase(),
+                })
+              : t("dialog.title.new", {
+                  what: t("terms.credential").toLowerCase(),
+                })
+          }
+          variant={ModalVariant.medium}
+          isOpen={isCreateUpdateModalOpen}
+          onClose={closeCreateUpdateModal}
+        >
+          <IdentityForm
+            identity={identityToUpdate ? identityToUpdate : undefined}
+            onClose={closeCreateUpdateModal}
+            xmlValidator={validateXML}
```

Comment:
Not needed

Reply:
I am sorry - we may still need this. Need to check with Jeff/Ramon here about support for mvn identities for mta 7. I Know we will need it for v0.2.0

Reply:
@ibolton336, this is out of scope for this PR. I haven't changed anything from previous modal. So let's open another issue if needed.